daniel.py

```diff
@@ -8,12 +8,12 @@
 if uro == ("tak"):
     kiedy_urodziny = x.year - wiek + 100
     kiedy_urodziny = str(kiedy_urodziny)
-    print("Super sprawa " + imię + " w " + kiedy_urodziny + " roku będziesz miał/a 100 lat")
+    print("Super sprawa. " + imię + " w " + kiedy_urodziny + " roku będziesz miał/a 100 lat")
 else:
     kiedy_urodziny = x.year - wiek + 99
     kiedy_urodziny = str(kiedy_urodziny)
     print("Super sprawa " + imię + " w " + kiedy_urodziny + " roku będziesz miał/a 100 lat")
```

Comment:
This code looks like a duplication of the code from lines 9-11. Can we extract it to a function?

Reply:
done